web/src/main/kotlin/no/nav/su/se/bakover/web/Extensions.kt

```diff
@@ -17,12 +18,27 @@ import no.nav.su.se.bakover.common.deserialize
 import no.nav.su.se.bakover.common.sikkerLogg
 import no.nav.su.se.bakover.domain.Fnr
 import no.nav.su.se.bakover.domain.Saksnummer
+import no.nav.su.se.bakover.web.features.suUserContext
 import java.util.UUID
 
-@Suppress("unused", "unused_parameter")
-internal fun ApplicationCall.audit(msg: String) {
-    // sikkerLogg.info("${suUserContext.navIdent} $msg")
-    sikkerLogg.info("Audit-logg fjernet, her må vi få på plass audit-logging til riktig sted!")
+internal fun ApplicationCall.sikkerlogg(msg: String) {
+    sikkerLogg.info("${suUserContext.navIdent} $msg") // TODO: Må se på om denne kan beholdes, avventer svar fra ATOM
```

Comment:
```suggestion
    sikkerLogg.info("${suUserContext.navIdent} $msg") 
```

Tror vi ihvertfall kan beholde den nå som vi har fjernet de to fnr-innslagene. Mulig at fnr også hadde vært greit, men der hadde ikke Atom blitt enige i møtet de hadde.